File: lista0/zadanie1.py
import math 
from scipy.integrate import quad

def y(n):
    lista = [math.log(6/5)]
    for i in range(1, n+1):
            lista.append(1/i - 5*lista[-1])
    return lista[n]


def calka(n):
    x_low = 0
    x_high = 1
    funkcja = lambda x: x**n/(x+5)
    return quad(funkcja, x_low, x_high)[0]


n = 24
for i in range(n+1):
    calka_wyn = calka(i)
    rekurencja = y(i)
    print(f"{i}: ", calka_wyn, rekurencja)

File: lista0/zadanie1.jl
using QuadGK

function y(n)
    lista = BigFloat[log(BigFloat(6)/5)]
    for i in 1:n
        append!(lista, 1/BigFloat(i) - 5*lista[end])
    end
    return lista[n+1]
end

function calka(n)
    x_low = 0
    x_high = 1
    funkcja = x -> x^n/(x+5)
    return quadgk(funkcja, x_low, x_high)[1]
end

n = 24
for i in 0:n
    calka_wyn = calka(i)
    rekurencja = y(i)
    println(calka_wyn, " ", rekurencja)
end


